src/App.tsx

```diff
@@ -17,6 +17,7 @@ const App = () => {
           <Route path="/tag" exact component={TagPage} />
           <Route path="/tag/add" exact component={Tag} />
           <Route path="/tag/:id/edit" exact component={Tag} />
+          {/* Doesn't this error without a passed in `contactId`? */}
```

Comment:
Because I was getting an error page on the default `/chat` route, I did a little digging into it, and I believe because contactId is never passed in for this route, the page will not render properly. So, I'm not completely sure how this worked before, or if I did something wrong on my end, but now to see the page properly, you need to add a ID at the end of the URL (i.e. localhost:3000/chat/2)